From bed6a4edc0691a8df82d0263c51cb6d062d3dad0 Mon Sep 17 00:00:00 2001
From: Shunsuke Kataoka <shunsuke.kataoka.df@renesas.com>
Date: Fri, 21 Jun 2013 14:37:11 +0900
Subject: [PATCH 656/715] ARM: shmobile: r8a7790: Add DVC route support

Add DVC route setting to r8a7790_audma_slaves and r8a7790_audmapp_slaves.

Signed-off-by: Shunsuke Kataoka <shunsuke.kataoka.df@renesas.com>
(cherry picked from commit f2b2600de790a3120249aae9cbbc0a8139632edc)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 arch/arm/mach-shmobile/setup-r8a7790.c |   10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/mach-shmobile/setup-r8a7790.c b/arch/arm/mach-shmobile/setup-r8a7790.c
index 1fd9423..1adca0a 100644
--- a/arch/arm/mach-shmobile/setup-r8a7790.c
+++ b/arch/arm/mach-shmobile/setup-r8a7790.c
@@ -951,6 +951,11 @@ static const struct sh_dmadesc_slave_config r8a7790_audma_slaves[] = {
 		.addr		= 0xec004400,
 		.chcr		= CHCR_RX(XMIT_SZ_32BIT),
 		.mid_rid	= 0x9c,
+	}, {
+		.slave_id	= SHDMA_SLAVE_PCM_CMD1_MEM,
+		.addr		= 0xec008400,
+		.chcr		= CHCR_RX(XMIT_SZ_32BIT),
+		.mid_rid	= 0xbe,
 	},
 };
 
@@ -961,6 +966,11 @@ static const struct sh_audmapp_slave_config r8a7790_audmapp_slaves[] = {
 		.dar		= 0xec400000,
 		.chcr		= 0x2d000000,
 	}, {
+		.slave_id	= SHDMA_SLAVE_PCM_CMD0_SSI0,
+		.sar		= 0xec308000,
+		.dar		= 0xec400000,
+		.chcr		= 0x37000000,
+	}, {
 		.slave_id	= SHDMA_SLAVE_PCM_SSI1_SRC1,
 		.sar		= 0xec401000,
 		.dar		= 0xec300400,
-- 
1.7.10.4

